app/hypnos/src/event_handler.c

```diff
@@ -143,13 +148,28 @@ void touch_tap_isr(struct device *touch_dev, struct sensor_trigger *tap)
 
 	sensor_channel_get(touch_dev, CST816S_CHAN_TOUCH_POINT_1, &touch_point);
 	LOG_INF("Gesture %d on x=%d, y=%d", gesture, touch_point.val1, touch_point.val2);
+	switch (gesture) {
+	case 1:
```

Comment:
should use the public API for that 
`cst816s_gesture` enum is there for that

Reply:
sure, fixed